app/models/organisation_brand_colour.rb

```diff
@@ -152,4 +152,9 @@ def self.find(class_name)
     title: "Department for International Trade",
     class_name: "department-for-international-trade",
   )
+  ForeignCommonwealthDevelopmentOffice = create!(
+    id: 30,
+    title: "Foreign & Commonwealth Development Office",
```

Comment:
```suggestion
    title: "Foreign, Commonwealth & Development Office",
```

Taken from https://www.gov.uk/government/organisations/foreign-commonwealth-development-office